src/wwwroot/services/auth.js

```diff
@@ -223,6 +225,14 @@
         method: 'GET',
         url: RELAY_CONFIG.baseUrl + '/accounts/' + getAccountName()  + '/status/limits'
       });
+    }   
+
+    function addFreeTrialNotificationToStorage(date) {
+      $window.localStorage.setItem('freeTrialNotificationOn', date);
+    }
+
+    function getFreeTrialNotificationFromStorage() {
+      return $window.localStorage.getItem('freeTrialNotificationOn');
```

Comment:
If I am not wrong, accessing to local storage has a cost, so, what do you think about read this value when the application is initialized and then use that value in place of read from local storage?